classes/query/Query.php

```diff
@@ -133,22 +133,23 @@ public function setIsClosed($isClosed)
      */
     public function getHeadNote()
     {
-        $notes = $this->getReplies(null, NoteDAO::NOTE_ORDER_DATE_CREATED, \PKP\db\DAO::SORT_DIRECTION_ASC);
-        return $notes->first();
+        return $this->getReplies(null, Note::NOTE_ORDER_DATE_CREATED)->first();
     }
 
     /**
      * Get all notes on a query.
      *
      * @param int $userId Optional user ID
-     * @param int $sortBy Optional NoteDAO::NOTE_ORDER_...
+     * @param int $sortBy Optional Note::NOTE_ORDER_...
      * @param int $sortOrder Optional DAO::SORT_DIRECTION_...
      *
      */
-    public function getReplies(?int $userId = null, int $sortBy = NoteDAO::NOTE_ORDER_ID, int $sortOrder = \PKP\db\DAO::SORT_DIRECTION_ASC): LazyCollection
+    public function getReplies(?int $userId = null, int $sortBy = Note::NOTE_ORDER_ID, int $sortOrder = DAO::SORT_DIRECTION_ASC): LazyCollection
```

Comment:
Hmm, I'm not 100% opposed to the current approach, but I think now that we're using Laravel scope functions for accessing notes we can get rid of `getReplies` and rely on `Note::withAssoc(...)->...`. Some callers want `withUser` and others not; some will benefit from lazy loading and others will just want the first note and that's all; etc. This function kind of emulates the old DAO-based functions we used to have with a list of fairly random parameters, and I'd rather we moved away from that.

Reply:
I've gone ahead and removed this and replaced the calls as suggested